tests/core/base/_utils/test_get_audio_type.py

```diff
@@ -0,0 +1,30 @@
+"""Tests the `_utils.get_audio_type` module."""
+
+import pytest
+
+from mirascope.core.base._utils.get_audio_type import get_audio_type
+
+
+def test_get_audio_type() -> None:
+    """Test the get_audio_type function."""
+    assert get_audio_type(WAV_DATA) == "wav"
+    assert get_audio_type(MP3_DATA) == "mp3"
+    assert get_audio_type(AIFF_DATA) == "aiff"
+    assert get_audio_type(AAC_DATA) == "aac"
+    assert get_audio_type(OGG_DATA) == "ogg"
+    assert get_audio_type(FLAC_DATA) == "flac"
+    with pytest.raises(ValueError, match="Unsupported audio type"):
+        get_audio_type(b"invalid")
+
+
+WAV_DATA = b"RIFF$\x00\x00\x00WAVEfmt \x10\x00\x00\x00\x01\x00\x01\x00D\xac\x00\x00\x88X\x01\x00\x02\x00\x10\x00data\x00\x00\x00\x00"
```

Comment:
any particular reason this is on bottom of file?

Reply:
Oops, I did this originally when I was using entire files, but they were way too large so I cut them down to single lines. Moved to the top of the file.